Cargo.toml

```diff
@@ -17,8 +18,7 @@ members = ["examples"]
 
 [features]
 default = ["async-tungstenite"]
-client-cynic = ["async-tungstenite", "cynic"]
-client-graphql-client = ["async-tungstenite", "graphql_client"]
```

Comment:
I would keep the features, it's much simpler for a user to enable `client-graphql-client` than having to figure out that you need `async-tungstenite` and `graphql_client`.

Reply:
Reverted.